Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Login & Signup Flow/NonprofitSignupViewController.m

```diff
@@ -61,31 +61,66 @@ -(void)saveNonprofitAndUserToParse {
     self.nonprofit.category = self.nonprofitCategoryTextField.text;
     self.nonprofit.websiteUrlString = self.nonprofitWebsiteTextField.text;
     self.user.nonprofit = self.nonprofit;
-    [self.user signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * _Nullable error) {
-        if (succeeded) {
-            if (self.finishedSavingBoth) {
-                [self performSegueWithIdentifier:@"loginSegue" sender:nil];
-            } else {
-                self.finishedSavingBoth = true;
-            }
+    
+    NSOperationQueue *opQueue = [NSOperationQueue new];
```

Comment:
Not 100% sure, but I think you might need to store this as a property, else it might get deallocated as soon as this method exists and operations may not run.